docs/basics/resources.rst

```diff
@@ -38,7 +38,7 @@ also composes the route matches and query parameters from the request. You may
 retrieve them from the event instance using the following methods:
 
 - ``getQueryParams()`` (returns a ``Zend\Stdlib\Parameters`` instance)
-- ``getRouteMatch()`` (returns a ``Zend\Mvc\Router\RouteMatch`` instance)
+- ``getRouteMatch()`` (returns a ``Zend\Router\RouteMatch`` instance)
```

Comment:
This will be either a `Zend\Router\RouteMatch` or a `Zend\Mvc\Rotuer\RouteMatch` instance, based on the version of zend-mvc used.